React/Views/RCTRefreshControl.m

```diff
@@ -33,6 +33,9 @@ - (void)layoutSubviews
 {
   [super layoutSubviews];
 
+  // Quick fix for iOS backgroundColor bug #7976 - To be removed on update
```

Comment:
Can you change this comment to look like
// Fix for bug #...
// TODO: Remove when updating to use iOS 10 refreshControl ScrollView prop.